typing/env.mli

```diff
@@ -373,6 +373,9 @@ val import_crcs: source:string -> crcs -> unit
 (* [is_imported_opaque md] returns true if [md] is an opaque imported module  *)
 val is_imported_opaque: modname -> bool
 
+(* [add_imported_opaque md] registers [md] as an opaque imported module  *)
+val add_imported_opaque: modname -> unit
```

Comment:
The switch from `add` in the function name to `register` in its description seems a bit worrying. Could align the name with the description, for instance with `register_as_opaque` ?

Reply:
I don't think that name is really any better. Adding something to the set opaque imported units vs. registering something as opaque is not a particularly useful distinction. I'd rather just delete the comment if it bothers you.

Reply:
Except that the very existence of a set of imported units is not exposed in the interface of `Persistent_env` so the function name refers to an implementation detail two module away.

Reply:
Fine, I have renamed it to `register_import_as_opaque`.